Lib/test/test_asyncore.py

```diff
@@ -433,7 +433,10 @@ def test_close_twice(self):
         f = asyncore.file_wrapper(fd)
         os.close(fd)
 
-        f.close()
+        os.close(f.fd)  # file_wrapper dupped fd
+        with self.assertRaises(OSError):
+            f.close()
+
         self.assertEqual(f.fd, -1)
```

Comment:
@haypo isn't this ^^^^ good enough?

Since file_wrapper.fd is not documented, maybe we should replace f.fd with f.fileno()? Do you want to add this change to the patch?